### PR TITLE
add new currency bob

### PR DESCRIPTION
**Pull request by emre-e**
[Related Issue](https://app.getsentry.com/udemycom/backenddjango/issues/114968687/)

### CODE REVIEW

**Comment by emre-e** [comment]
@udemy/payments can you guys review?
Thanks :tada:

File: moneyed/localization.py [hunk 1]
```diff
@@ -197,6 +197,7 @@ def format(self, money, include_symbol=True, locale=DEFAULT,
 _sign(DEFAULT, moneyed.BIF, prefix='FBu')
 _sign(DEFAULT, moneyed.BMD, prefix='BD$')
 _sign(DEFAULT, moneyed.BND, prefix='B$')
+_sign(DEFAULT, moneyed.BOB, suffix=' Bs')
```
Comment (sryavuz) [inline moneyed/localization.py:200]: There is no space for other currencies at the beginning. Is there something exceptional for BOB?

Comment (emre-e) [inline moneyed/localization.py:200]: The difference is this is a suffix sign. Most of the other suffixes has a space like this one.

``` python
_sign(DEFAULT, moneyed.DKK, suffix=' Dkr')
_sign(DEFAULT, moneyed.EUR, suffix=' €')
...
```

Comment (sryavuz) [inline moneyed/localization.py:200]: Ohh, ok :) I just saw `_sign(DEFAULT, moneyed.AFN, suffix='؋')`

**Comment by sryavuz** [comment]
Lgtm :+1:

**Comment by burakgoynuk** [comment]
LGTM

**Comment by dirtybit** [comment]
LGTM

**Pull request merged**